test/CMakeLists.txt

```diff
@@ -120,9 +120,10 @@ if(ENABLE_PYTHON_BINDINGS)
       python_valhalla
     VERBATIM)
   add_custom_target(run-python_valhalla DEPENDS python_valhalla.log)
+  set(python_tests python_valhalla)
 endif()
 
 ## High-level targets
-string(REGEX REPLACE "([^;]+)" "run-\\1" test_targets "${tests};${cost_tests};python_valhalla")
+string(REGEX REPLACE "([^;]+)" "run-\\1" test_targets "${tests};${cost_tests};${python_tests}")
```

Comment:
So, no pythons tests if the data tools aren't built?

Doesn't that mean we can build un-tested python bindings?

Reply:
correct. its because the python test in this case relies on being able to create valhalla tiles (since it tests real data). i dont see any other way around this. the only other option is to change the test to test less of the functionality or to checking static tiles (yuck).

Reply:
I guess, philosophically, the python binding tests should be testing the *binding* part, not necessarily what's being bound to.  That would indicate that backend stubs/mocks would work here.  However, those would need to be built.  As long as we have a CI run that enables both and tests the interfaces, I think we're fine.